src/index.html

```diff
@@ -1,13 +1,60 @@
-<!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
-</html>
+  <!doctype html>
+  <html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+          href="https://fonts.gstatic.com"
+          crossorigin=""
+        >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles/index.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img class="card__picture"
+           src="images/imac.jpeg"
+           alt="Imac"
+      >
+      <p class="card__title">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </p>
+
+      <p class="card__code">
+        Код товара: 195434
+      </p>
+
+      <div class="rating card__rating">
+        <div class="stars rating__stars_stars-value--4">
```

Comment:
```suggestion
        <div class="rating__stars stars stars--4">
```
rating__stars - элемент внешнего блока
stars - блок
stars--4 - модификатор блока